test/plato_test.js

```diff
@@ -130,5 +129,18 @@ exports['plato'] = {
       test.ok(overview.summary.total.jshint === 2, 'Should contain total jshint issues');
       test.done();
     });
+  },
+  'test file ending checks' : function(test) {
+    var files = [
+      'test/fixtures/test.es'
+    ];
+    
+    text.expect(1);
```

Comment:
Sorry for my bad english.

I would like to say that you did a mistake. On line 138, you wrote the word "text" instead of the word "test". That's why the tests don't passed.